trunk/SUAVE/Components/Energy/Storages/Batteries/Constant_Mass/Lithium_Ion_LiNCA_18650.py

```diff
@@ -0,0 +1,349 @@
+        
+        Properties Used:
+        N/A
+        
+    """   
+    def __defaults__(self):    
+        self.tag                              = 'Lithium_Ion_LiNCA_Cell' 
+                                             
+        self.cell.diameter                    = 0.0184                                                   # [m]
+        self.cell.height                      = 0.0652                                                   # [m]
+        self.cell.mass                        = 0.048 * Units.kg                                         # [kg]
+        self.cell.surface_area                = (np.pi*self.cell.height*self.cell.diameter) + (0.5*np.pi*self.cell.diameter**2)  # [m^2]
+        self.cell.volume                      = np.pi*(0.5*self.cell.diameter)**2*self.cell.height 
+        self.cell.density                     = self.cell.mass/self.cell.volume                          # [kg/m^3]  
+        self.cell.electrode_area              = 0.0342                                                   # [m^2] 
+                                                                                                 
+        self.cell.max_voltage                 = 4.2                                                      # [V]
+        self.cell.nominal_capacity            = 3.30                                                     # [Amp-Hrs]
+        self.cell.nominal_voltage             = 3.6                                                      # [V]
+        self.cell.charging_voltage            = self.cell.nominal_voltage                                # [V]  
+        
+        self.watt_hour_rating                 = self.cell.nominal_capacity  * self.cell.nominal_voltage  # [Watt-hours]      
+        self.specific_energy                  = self.watt_hour_rating*Units.Wh/self.cell.mass            # [J/kg]
+        self.specific_power                   = self.specific_energy/self.cell.nominal_capacity          # [W/kg]   
+        self.resistance                       = 0.025                                                    # [Ohms]
+                                                                                               
+        self.specific_heat_capacity           = 837.4                                                    # [J/kgK] 
+        self.cell.specific_heat_capacity      = 837.4                                                    # [J/kgK]   
+        self.cell.radial_thermal_conductivity = 0.8                                                      # [J/kgK]  
+        self.cell.axial_thermal_conductivity  = 32.2                                                     # [J/kgK]  
+            
+        battery_raw_data                      = load_NCA_raw_results()                                                   
+        self.discharge_performance_map        = create_discharge_performance_map(battery_raw_data)        
+        return  
+    
+    
+    def energy_cycle_model(self,numerics,discharge_flag = True ):  
```

Comment:
I feel like these scripts (different battery chemistry) can be condensed some

Reply:
I want to keep them separate because I will be building upon these in the future